templates/pages/cv_entry.md

```diff
@@ -1,5 +1,5 @@
 # {{ entry.system.electrodes.working_electrode.material }}({{ entry.system.electrodes.working_electrode.crystallographic_orientation }}) <small>- {{ entry.system.electrolyte | render("components/electrolyte.md") }}</small>
-<small>echemdb identifier: `{{ entry.identifier }}`</small>\
+<small>echemdb identifier: `{{ entry.identifier }}`</small>  
```

Comment:
@saraedum  Somehow `\` is not recognized by our mkdocs configuration as a new line. Hence I added two spaces which is equivalent in MD.